contentcuration/contentcuration/static/js/edit_channel/tree_edit/views.js

```diff
@@ -546,7 +546,11 @@ var ContentItem = BaseViews.BaseWorkspaceListNodeItemView.extend({
 	},
 	make_inline_copy: function(event) {
 		this.cancel_actions(event);
-		this.make_copy();
+		if(this.model.has_related_content()){
+			dialog.alert(this.get_translation("warning"), this.get_translation("related_content_alert"), this.make_copy);
```

Comment:
If a contentnode has prerequisites, it won't be copied when duplicated. This particular function is for copying inline, which wouldn't cause any problems with broken relationships, but there were some weird bugs around setting the prerequisites, so I'm handling it the same as copying to the clipboard for now. When it's copied to the clipboard, this is to prevent content from referencing any items that might not be available on Kolibri (e.g. Channel A has Video 1, which has Video 2 as a prerequisite. Video 1 from  is copied to the clipboard then moved to Channel B. Channel B is published, but Video 2 doesn't exist on Channel B, so the relationship would be broken).